tests/test_visitors/test_ast/test_statements/test_statements_with_no_effect.py

```diff
@@ -193,6 +194,31 @@ def test_statement_with_no_effect(
     assert_errors(visitor, [StatementHasNoEffectViolation])
 
 
+@pytest.mark.parametrize('code', [
+    module_template,
+])
+@pytest.mark.parametrize('statement', [
+    'x += x + 2',
+    'x -= x - 1',
+    'x *= x * 1',
+    'x /= x / 1',
```

Comment:
Still not all BinOps are covered. Please, see the list:

- `-=`
- `*=`
- `**=`
- `/=`
- `^=`
- `%=`
- `>>=`
- `<<=`
- `&=`
- `|=`

Reply:
added more tests